src/components/templates/PosterPage/components/ShareModal/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { FACEBOOK_APP_ID } from "secrets";
+import { PosterPageVenue } from "types/venues";
+import { WithId } from "utils/id";
+
+const getCategoriesFromVenue = (venue: WithId<PosterPageVenue>) => {
+  return venue?.poster?.categories || [];
+};
```

Comment:
Why do we need this function? This seems like something we would just tend to do directly?

Assuming we need/want it, this aligns with our standards + fixes unneeded safety checks/etc:
```suggestion
const getCategoriesFromVenue = (venue: WithId<PosterPageVenue>) => venue.poster?.categories ?? [];
```

Reply:
https://github.com/sparkletown/sparkle/pull/1396/commits/a0cb5302188a3401cc34dbca148d6870a0e629ea